helm_deploy/values-dev.yaml

```diff
@@ -9,7 +9,7 @@ generic-service:
     OAUTH_ENDPOINT_URL: https://sign-in-dev.hmpps.service.justice.gov.uk/auth
     COMMUNITY_ENDPOINT_URL: https://community-api.test.probation.service.justice.gov.uk/secure
     ASSESSMENT_ENDPOINT_URL: https://offender-dev.aks-dev-1.studio-hosting.service.justice.gov.uk
-
+    HMPPS_TIER_ENDPOINT_URL: https://hmpps-tier-dev.hmpps.service.justice.gov.uk
```

Comment:
you can use variables in Helm to define the url in one place.

To set a variable https://github.com/ministryofjustice/hmpps-workload/blob/main/helm_deploy/values-dev.yaml#L7

To reference a variable https://github.com/ministryofjustice/hmpps-workload/blob/main/helm_deploy/values-dev.yaml#L18